kuma/javascript/src/payments/components/subscription-form.jsx

```diff
@@ -199,6 +165,7 @@ export default function SubscriptionForm() {
                         createSubscription();
                     },
                     closed() {
+                        setOpenStripeModal(false);
```

Comment:
This is the line that fixes the ticket (https://github.com/mdn/kuma/issues/6931)